packages/js-sdk/src/datastore/cachestore.ts

```diff
@@ -171,6 +172,36 @@ export class CacheStore {
     return cachedDoc;
   }
 
+  async create(docs: any, options: any = {}) {
+    if (!isArray(docs)) {
+      return this.createOne(docs, options);
+    }
+
+    const apiVersion = getApiVersion();
+    if (apiVersion < 5) {
+      throw new KinveyError('Unable to create an array of entities. Please create entities one by one or use API version 5 or newer.');
+    }
+
+    const createManyResult = {
+      entities: new Array(docs.length).fill(null),
+      errors: []
+    };
+
+    const createPromises = docs.map((doc, index) => {
+      return this.createOne(doc, options)
```

Comment:
This could be pretty slow if things still work the way I remember them. There's a `createOne` but no `createMany`?

Reply:
Well, this doesn't improve things but doesn't make them worse either. This change is meant to unify the data store APIs, so all of them have a `create` method that accepts an array. There isn't much we can do for the cache store besides iterating the entities.